journal_brief/cli/main.py

```diff
@@ -17,17 +17,25 @@
 """
 
 import argparse
+from email.mime.text import MIMEText
+from email import charset
+import io
 from locale import setlocale, LC_ALL
 import logging
 import os
 import signal
+import smtplib
+import ssl
+import subprocess
 import sys
 
 from journal_brief import (SelectiveReader,
                            LatestJournalEntries,
                            get_formatter,
                            list_formatters,
                            JournalFilter)
+from journal_brief.cli.constants import EMAIL_SUPPRESS_EMPTY_TEXT
+from journal_brief.cli.constants import EMAIL_DRY_RUN_SEPARATOR
```

Comment:
Minor nit: could use `from ... import (..., ...)` here (and other places we import these).

Reply:
Done.